volpe/builder.py

```diff
@@ -113,6 +113,8 @@ def number_iter(self, tree: TypeTree):
             start = read_environment(b, args[0], env_types)[0]
             b.ret(b.add(start, args[1]))
 
+        # TODO going in reverse
```

Comment:
what is this about?

Reply:
`0..10` gives 
```
i => {i >= 0; i < 10; :> i}
```

I want `10..0` to give
```
i => {i >= 0; i < 10; :> 9 - i} 
# 9 and not 10, because I want 10..0 to be reverse of 0..10
```

This is not related to the pretty return, so I can remove it.

Reply:
ah makes sense